R/ts_plot.R

```diff
@@ -1,19 +1,38 @@
+library(ggplot2)
+library(forecast)
 #' Time series decomposition and plot
 #'
 #' Convert csv file into a time series object, decompose it into in trend, seasonality/cyclicity
 #' and noise/remainder/error and plot the raw data and the decomposed components.
 #'
 #' @param data str: the path of the csv file
-#' @param type str: "regular" or "irregular"
-#'            "regular": time series with regular time intervals
-#'            "irregular": time series with irregular time intervals
 #' @param col str: the column to be analyzed
 #' @param frequency int: the desired time interval
 #'
-#' @return a figure with 4 subplots
+#' @return ggplot Object
 #' @export
 #' @examples
-#' ts_plot("https://raw.githubusercontent.com/plotly/datasets/master/timeseries.csv", A, irregular, 3)
-ts_plot <- function(data, col, type, frequency){
-  print("hello")
+#' ts_plot("~/Desktop/ts4_jjshares.csv", "earnings", 4)
```

Comment:
Maybe replace with a built in dataset so that any user could replicate this example

Reply:
Since it is the path of  a csv file, it is hard to use a built-in dataset...